frontend/src/views/EditRoom.jsx

```diff
@@ -96,11 +98,16 @@ function EditRoom() {
       flexDirection: 'column',
     }}
     >
-      <Typography component="h1" variant="h4" sx={{ textAlign: 'center' }}>
-        Edit Room
-        {' '}
-        {roomId}
-      </Typography>
```

Comment:
I think it would be better to use JavaScript code inside the Typography component to generate dynamic content instead of separating the content using {' '}

Reply:
Done